src/backend/distributed/utils/background_jobs.c

```diff
@@ -558,6 +522,43 @@ CitusBackgroundTaskQueueMonitorMain(Datum arg)
 }
 
 
+/*
+ * CalculateBackoffDelay calculates the time to backoff between retries.
+ *
+ * Per try we increase the delay as follows:
+ *   retry 1: 1 min
+ *   retry 2: 10 min
+ *   retry 3: 60 min
+ *
+ * In the future we would like a callback on the job_type that could
+ * distinguish the retry count and delay + potential jitter on a
```

Comment:
Don't we want jitter for the rebalancer?

Reply:
jitter is mainly for when there are multiple actors competing for resources

Reply:
https://github.com/citusdata/citus/pull/6296#discussion_r962725306